src/emu/emuopts.cpp

```diff
@@ -396,9 +1045,11 @@ std::string slot_option::specified_value() const
 
 void slot_option::specify(std::string &&text)
 {
+	// record the old value; we may need to trigger an update
+	std::string old_value = value();
```

Comment:
Why didn't you make this `const`?  It's a saved value that shouldn't be messed with.  Same for the next one below.